streams/src/main/java/org/apache/kafka/streams/kstream/internals/CogroupedStreamAggregateBuilder.java

```diff
@@ -31,90 +26,90 @@
         this.builder = builder;
     }
 
-    <KR, VIn, W extends Window> KTable<KR, VOut> build(
-            final Map<KGroupedStreamImpl<K, ?>, Aggregator<? super K, ? super Object, VOut>> groupPatterns,
-            final Initializer<VOut> initializer,
-            final NamedInternal named,
-            final StoreBuilder<? extends StateStore> storeBuilder,
-            final Serde<KR> keySerde,
-            final Serde<VOut> valSerde,
-            final String queryableName,
-            final Windows<W> windows,
-            final SessionWindows sessionWindows,
-            final Merger<? super K, VOut> sessionMerger) {
+    <KR, VIn, W extends Window> KTable<KR, VOut> build(final Map<KGroupedStreamImpl<K, ?>, Aggregator<? super K, ? super Object, VOut>> groupPatterns,
+                                                       final Initializer<VOut> initializer,
+                                                       final NamedInternal named,
+                                                       final StoreBuilder<? extends StateStore> storeBuilder,
+                                                       final Serde<KR> keySerde,
+                                                       final Serde<VOut> valSerde,
+                                                       final String queryableName,
+                                                       final Windows<W> windows,
+                                                       final SessionWindows sessionWindows,
+                                                       final Merger<? super K, VOut> sessionMerger) {
 
         final Collection<? extends AbstractStream<K, ?>> groupedStreams = new ArrayList<>(groupPatterns.keySet());
         final AbstractStream<K, ?> kGrouped = groupedStreams.iterator().next();
         groupedStreams.remove(kGrouped);
         kGrouped.ensureCopartitionWith(groupedStreams);
 
-
         final Collection<StreamsGraphNode> processors = new ArrayList<>();
         boolean stateCreated = false;
         int counter = 0;
-        for (final Entry<KGroupedStreamImpl<K, ?>, Aggregator<? super K, ? super Object, VOut>> kGroupedStream : groupPatterns
-                .entrySet()) {
+        for (final Entry<KGroupedStreamImpl<K, ?>, Aggregator<? super K, ? super Object, VOut>> kGroupedStream : groupPatterns.entrySet()) {
             final StatefulProcessorNode statefulProcessorNode = getStatefulProcessorNode(
-                    kGroupedStream.getValue(),
-                    initializer,
-                    named.withName(named.suffixWithOrElseGet("-cogroup-agg-" + counter++, builder, CogroupedKStreamImpl.AGGREGATE_NAME)),
-                    stateCreated,
-                    storeBuilder,
-                    windows,
-                    sessionWindows,
-                    sessionMerger);
+                kGroupedStream.getValue(),
+                initializer,
+                named.suffixWithOrElseGet(
```

Comment:
No need to pass in a `Named` -- we can just pass in the actual name as `String` directly -- otherwise we call `suffixWithOrElseGet` twice for no reason